css-display-3/Overview.bs

```diff
@@ -75,15 +75,15 @@ Introduction</h2>
 
 	<p><em>This section is normative.</em>
 
-	CSS takes a source document organized as
+	CSS takes a <a>document tree</a> organized as
```

Comment:
This change makes the sentence somewhat redundant? We _are_ conceptually working with a source document, even if the source document has its own processing model and processed into an _element tree_ by the time CSS looks at it. And that's what this sentence is trying to explain.

So I would not make this change.